src/localstorage/localStorage.js

```diff
@@ -0,0 +1,31 @@
+import { quizData } from "../data.js";
+export const addStorage = (question) => {
```

Comment:
You are getting here the whole quizData, why don't just store it as it is in localStorage, instead of pushing it inside an array?